### PR TITLE
Reformat order controllers

### PR DESCRIPTION
**Pull request by z0723-julie**

### CODE REVIEW

**Review by darylhjd — CHANGES_REQUESTED** [review]
Make each choice its own method. Also, use `InputParser` methods for getting input, no need to write your own.

**Review by darylhjd — APPROVED** [review]

File: src/main/java/seedu/duke/controllers/OrderController.java [hunk 1]
```diff
@@ -12,13 +13,13 @@
...
-    public OrderController(Scanner scanner) {
-        super(CHOICES, scanner);
+    public OrderController() {
+        super(CHOICES);
         orderManager = new OrderManager();
     }
 
     @Override
-    protected boolean optionSwitcher(int choice) throws IllegalArgumentException {
+    protected boolean optionSwitcher(int choice) throws IllegalArgumentException, OperationTerminationException {
```
Comment (BotBw) [inline src/main/java/seedu/duke/controllers/OrderController.java:22]: Only `OperationTerminationException` here

File: src/main/java/seedu/duke/controllers/OrderController.java [hunk 2]
```diff
@@ -12,13 +13,13 @@
...
         switch (choice) {
         case 0:
```
Comment (BotBw) [inline src/main/java/seedu/duke/controllers/OrderController.java:24]: return true here?

File: src/main/java/seedu/duke/controllers/OrderController.java [hunk 3]
```diff
@@ -73,6 +52,41 @@ protected boolean optionSwitcher(int choice) throws IllegalArgumentException {
         return false;
     }
 
+    private void quitOrder() throws OperationTerminationException {
```
Comment (BotBw) [inline src/main/java/seedu/duke/controllers/OrderController.java:55]: this should be something like `addMultipleDish`?

**Pull request merged**